yoyo.ini

```diff
@@ -0,0 +1,8 @@
+[DEFAULT]
+sources = %(PATH_SOURCES)s
+database = postgresql://%(DB_LOGIN)s:%(DB_PASSWORD)s@%(CONTAINER_NAME)s:%(CONTAINER_PORT)s/%(DB_NAME)s
+migration_table = _yoyo_migration
+batch_mode = on
+verbosity = 1
+post_create_command = hg add {}
```

Comment:
Why do we need it?

Reply:
I haven't figured it out completely, I'll write an answer later.

Reply:
And where is the answer?